makefile

```diff
@@ -0,0 +1,25 @@
+SHELL := /bin/bash
+MODULE=weevenetwork/translator
```

Comment:
MODULE variable value must be the same as `image: name: weevenetwork/generic-translator` in Module.yaml or it's gonna publish to a wrong repo

Reply:
updated